src/content/fr/2019/caching.md

```diff
@@ -0,0 +1,691 @@
+  </a>
+  <div id="fig2-description" class="visually-hidden">Ces deux diagrammes à barres identiques pour le mobile et les ordinateurs de bureau montrent que 72&nbsp;% des requêtes utilisent des en-têtes Cache-Control et que 56&nbsp;% utilisent Expires et les 27&nbsp;% n'utilisent aucun des deux.</div>
+  <figcaption id="fig2-caption">Figure 2. Présence des en-tête HTTP <code>Cache-Control</code> et <code>Expires</code>.</figcaption>
+</figure>
+
+## Quel type de contenu met-on en cache&nbsp;?
+
+Une ressource mise en cache est stockée par le client pendant un certain temps et peut être réutilisée ultérieurement. Pour les requêtes HTTP, 80&nbsp;% des réponses peuvent certainement être mises en cache, ce qui signifie qu'un système de cache peut les stocker. En dehors de ça,
+
+*   6&nbsp;% des requêtes ont un <i lang="en">Time To Live</i> (TTL) de 0 seconde, qui invalide immédiatement une entrée en cache.
+*   27&nbsp;% sont mis en cache par heuristique, à cause d'un `Cache-Control` manquant en en-tête.
+*   47&nbsp;% sont mis en cache pendant plus de 0 seconde.
+
+Les autres réponses ne peuvent pas être stockées dans le cache du navigateur.
+
+<figure>
+  <a href="/static/images/2019/16_Caching/fig3.png">
+    <img src="/static/images/2019/16_Caching/fig3.png" alt="Figure 3. Distribution des réponses pouvant être mises en cache." aria-labelledby="fig3-caption" aria-describedby="fig3-description" width="600" height="371" data-width="600" data-height="371" data-seamless data-frameborder="0" data-scrolling="no" data-iframe="https://docs.google.com/spreadsheets/d/e/2PACX-1vT3GWCs19Wq0mu0zgIlKRc8zcXgmVEk2xFHuzZACiWVtqOv8FO5gfHwBxa0mhU6O9TBY8ODdN4Zjd_O/pubchart?oid=1868559586&amp;format=interactive">
+  </a>
+  <div id="fig3-description" class="visually-hidden">Un graphique à barres superposées montrant que 20&nbsp;% des réponses pour ordinateurs de bureau ne peuvent être mises en cache, 47&nbsp;% ont un cache supérieur à zéro, 27&nbsp;% sont mises en cache de manière heuristique et 6&nbsp;% ont un TTL de 0. Les statistiques pour les mobiles sont très similaires (19&nbsp;%, 47&nbsp;%, 27&nbsp;% et 7&nbsp;%)</div>
+  <figcaption id="fig3-caption">Figure 3. Distribution des réponses pouvant être mises en cache.</figcaption>
+</figure>
+
+Le tableau ci-dessous détaille les [TTL](https://fr.wikipedia.org/wiki/Time_to_Live) du cache pour les requêtes en provenance d'ordinateurs de bureau. La plupart des types de contenu sont mis en cache, mais les ressources CSS semblent toujours être mises en cache à des valeurs TTL élevées.
```

Comment:
Doesn't look like my previous comment was addressed here:

I see you've included a wikipedia reference here thats not in the English version. If you think things like this are valuable then I would encourage you to also include such changes to the English version i) to keep them in synch and ii) to benefit English speakers. Similarly if you spot typos while translating.

However in this specific case I would question its value. The [French version](https://fr.wikipedia.org/wiki/Time_to_Live) talks about IP TTL and DNS TTL, but not HTTP TTL which we are discussing here. Funnily enough, [the English version of that page](https://en.wikipedia.org/wiki/Time_to_live#HTTP) _does_ talk about HTTP TTL but even then only covers it very briefly so not sure if that link is more confusing than beneficial?

To be honest I'd remove this. However if you really want it, then a better link might be the MDN docs (in [French](https://developer.mozilla.org/fr/docs/Glossaire/TTL#Mise_en_cache) and [English](https://developer.mozilla.org/en-US/docs/Glossary/TTL#Caching))?

```suggestion
Le tableau ci-dessous détaille les TTL du cache pour les requêtes en provenance d'ordinateurs de bureau. La plupart des types de contenu sont mis en cache, mais les ressources CSS semblent toujours être mises en cache à des valeurs TTL élevées.
```